cmd/ocm-backplane/managedJob/createManagedJob.go

```diff
@@ -22,112 +36,182 @@ func newCreateManagedJobCmd() *cobra.Command {
 		RunE:          runCreateManagedJob,
 	}
 
-	cmd.Flags().StringArrayP(
+	cmd.Flags().StringArrayVarP(
+		&options.params,
 		"params",
 		"p",
 		[]string{},
 		"Params to be passed to managedjob execution in json format. For e.g. -p 'VAR1=VAL1' -p VAR2=VAL2 ")
 
+	cmd.Flags().BoolVarP(
+		&options.wait,
+		"wait",
+		"w",
+		false,
+		"Wait until command execution is finished")
+
 	return cmd
 }
 
-func runCreateManagedJob(cmd *cobra.Command, args []string) error {
-	// ======== Parsing Flags ========
-	// Params flag
-	arr, err := cmd.Flags().GetStringArray("params")
+// runCreateManagedJob creates managed job in the specific cluster
+func runCreateManagedJob(cmd *cobra.Command, args []string) (err error) {
+
+	// init params and validate
+	err = initParams(cmd, args)
+
 	if err != nil {
 		return err
 	}
 
-	parsedParams, err := utils.ParseParamsFlag(arr)
+	// ======== Initialise backplaneURL ========
```

Comment:
```suggestion
	// ======== Initialize backplaneURL ========
```

Reply:
right ..time to change my British EN lang pack to USA EN lang pack :)